index.js

```diff
@@ -0,0 +1,14 @@
+require('babel-core/register')
+require('coffee-script/register')
+
+const env = require('node-env-file')
+
+if (process.env.NODE_ENV === 'test') {
+  env('./.env.test')
+
+  // Other envs: development, staging, production
+} else {
+  env('./.env')
+}
+
+require('./boot')
```

Comment:
Created this boot loader so that from the point of ./app onwards we can write ES6.